annotation/util.scm

```diff
@@ -57,7 +57,7 @@
    (make-edge (make-edge-info node2 node1 name pubmedId subgroup annotation) "edges")
 )
 
-;; Find node name and description
+;; Find node name and description. (What's a node ???)
```

Comment:
Atoms are nodes and Links are edges as of the Json parser.

Reply:
Any atom is a json-node? Can anything "have a name", or just certain things, like, say, organelles? What are the legal arguments for this function?

Reply:
I can say any bio atom which needs to be visualized is a json-node regardless of their type.  genes, proteins/Small molecules, RNA's, GO terms, cellular location concept nodes ...

Reply:
OK, pushed a patch to resolve this. I called it an "entity", because a "node" is alreay a reserved term for the atomspace...